README.md

```diff
@@ -201,6 +201,20 @@ class SomeClass
 }
 ```
 
+### Filter Rectors
+
+If you have a configuration file for Rector including many sets and Rectors, you might want at times to run only a single
+Rector from them. The `--only` argument allows that, for example : 
```

Comment:
Here is extra space in sentence. It might create wierd spacing.
Better let Markdown/IDE handle that